templates/gallery/partials/landing-images.snip.html

```diff
@@ -0,0 +1,42 @@
+{{!
+<!---
+Copyright 2017 The AMP Start Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+}}
+
+{{#landing-images}}
```

Comment:
To me this looks exactly like "image-with-caption", any blockers in not being able to reuse? 

Remember you can always do something like 

`<section class=""namespace-sectionname> (or) <div>` and the target the ampstart- classes inside them to a different style.

You will have to lookout for semantics & accessibility (you don't have to worry about this if you reuse)

Reply:
You're right that it is really similar, but we really need the `{{#positions}}` classes injected into the `<figcaption>` element, and that isn't doable with the "image-with-caption" component like it is with our custom code.

Reply:
if that is the only reason , then it is easy to solve this

0. Use nth child - IMO having numbered classes is really a bad idea.
1. Pass in the full class via JSON (add a fig caption class inside the image component)

Reply:
@camelburrito Beyond the fact that the markup contains similar tags (not even in the same structure) this shares little in common with the `image-with-caption` component.

Switching to `image-with-caption` would mean all of the styles would have to be re-written to adapt to the new markup structure. The current structure is directly coupled to the interactive animations and image transformations. There is a risk that the markup structure of `image-with-caption` entirely incompatible with the desired effects.

Reply:
I see, that makes some sense , lets proceed with this then